src/core_plugins/i18n/server/__tests__/i18n.js

```diff
@@ -0,0 +1,214 @@
+import expect from 'expect.js';
+import _ from 'lodash';
+import { join } from 'path';
+
+import * as i18n from '../i18n/i18n';
+
+const FIXTURES = join(__dirname, 'fixtures');
+
+describe('i18n module', function () {
+
+  describe('one plugin', function () {
+
+    beforeEach('registerTranslations - one plugin', function () {
+      const pluginName = 'test_plugin_1';
+      const pluginTranslationPath = join(FIXTURES, 'translations', pluginName);
+      const translationFiles = [
+        join(pluginTranslationPath, 'de.json'),
+        join(pluginTranslationPath, 'en.json')
+      ];
+      translationFiles.forEach(i18n.registerTranslations);
+    });
+
+    afterEach('unregisterTranslations - one plugin', function () {
+      i18n.unregisterTranslations();
+    });
+
+    describe('getTranslationsForLocale', function () {
+
+      it('should return the translations for en locale as registered' , function () {
+        const languageTag = 'en';
+        const expectedTranslationJson = {
+          'test_plugin_1-NO_SSL': 'Dont run the dev server using HTTPS',
+          'test_plugin_1-DEV': 'Run the server with development mode defaults',
+          'test_plugin_1-NO_RUN_SERVER': 'Dont run the dev server',
+          'test_plugin_1-HOME': 'Run along home now!'
+        };
+        return checkTranslations(expectedTranslationJson, languageTag);
+      });
+
+      it('should return the translations for de locale as registered' , function () {
+        const languageTag = 'de';
+        const expectedTranslationJson = {
+          'test_plugin_1-NO_SSL': 'Dont run the DE dev server using HTTPS',
+          'test_plugin_1-DEV': 'Run the DE server with development mode defaults'
+        };
+        return checkTranslations(expectedTranslationJson, languageTag);
+      });
+
+      it('should return translations for default locale which is set to the en locale' , function () {
+        const expectedTranslations = {
+          'test_plugin_1-NO_SSL': 'Dont run the dev server using HTTPS',
+          'test_plugin_1-DEV': 'Run the server with development mode defaults',
+          'test_plugin_1-NO_RUN_SERVER': 'Dont run the dev server',
+          'test_plugin_1-HOME': 'Run along home now!'
+        };
+        i18n.setDefaultLocale('en');
+        return checkTranslationsForDefaultLocale(expectedTranslations);
```

Comment:
Could you pull this and other tests for `getTranslationsForDefaultLocale` into a separate describe block specifically for that method?